packages/editor/src/components/post-preview-button/index.js

```diff
@@ -157,7 +158,7 @@ export default compose( [
 			isDirty: isEditedPostDirty(),
 			isNew: isEditedPostNew(),
 			isSaveable: isEditedPostSaveable(),
-			isAutosaveable: isEditedPostAutosaveable(),
+			isEditedPostAutosaveable: isEditedPostAutosaveable,
```

Comment:
This is confusing, why do we pass down selector rather than calling it here?

Reply:
> This is confusing, why does it pass down selector rather than calling it here?

From original comment:

>I'm also not sure if it would be a good idea to pass the selector function as a prop instead of the value, but like that we can call it only when needed.

It is a bit strange, though I understand why it's being done. I think though that if we were to go this direction, I'd rather we move toward dropping all consideration of whether the post is autosaveable, and just use the "is dirty" with error tolerance from the autosave REST response for unneeded saves.

Reply:
Either way, we should document the solution to ensure that it doesn't get refactor by accident.